pre-cache/Containerfile

```diff
@@ -0,0 +1,6 @@
+FROM registry.access.redhat.com/ubi8-minimal:latest
```

Comment:
Since this will be bundled with the operator, we should just include under the existing Containerfile